salt/modules/win_iis.py

```diff
@@ -1255,6 +1255,54 @@ def set_container_setting(name, container, settings):
         salt '*' win_iis.set_container_setting name='MyTestPool' container='AppPools'
             settings="{'managedPipeLineMode': 'Integrated'}"
     '''
+
+    def identity_type_map(identity_id,map_type):
+        '''
+
+        :param identity_id: app pool identity type id - string or string number   
+        :return: the equivalent id number ot string
+         mapping:
+                'LocalSystem' <--> '0'
+                'LocalService' <--> '1'
+                'NetworkService' <--> '2'
+                'SpecificUser' <--> '3'
+                'ApplicationPoolIdentity' <--> '4'
+        '''
+
+        available_ids = ('0','1','2','3','4','LocalSystem','LocalSystem','NetworkService','SpecificUser','ApplicationPoolIdentity')
```

Comment:
Please place spaces between items in the list.
Second string item should be `LocalService`
Also, wrap after 80 characters:
```
        available_ids = ('0', '1', '2', '3', '4', 'LocalSystem', 'LocalService',
                         'NetworkService', 'SpecificUser',
                         'ApplicationPoolIdentity')
```